test/unit/models/qrCodeAuth.test.js

```diff
@@ -3,16 +3,51 @@ const { expect } = chai;
 
 const cleanDb = require("../../utils/cleanDb");
 const firestore = require("../../../utils/firestore");
-const qrCodeAuthModel = require("../../../models/qrCodeAuth");
-const qrCodeAuth = firestore.collection("QrCodeAuth");
+// const userDeviceInfo = require("../../../models/qrCodeAuth");
+// const userDeviceInfoModel = firestore.collection("QrCodeAuth");
```

Comment:
can we remove the commented code if its not needed

Reply:
removed it